src/stylesheets/components/_accordions.scss

```diff
@@ -90,3 +90,7 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-bottom: 0;
   }
 }
+
+.usa-accordion-content-collapsed {
```

Comment:
@msecret - Interesting suggestion. I'm not sure I'm seeing the utility solution that you might be. If we make a `mixin` for `display: none`, wouldn't we still need to `@include` it somewhere that distinguishes the two states? 

@maya - Seems an interesting alternative. After looking at the JavaScript for the Frend accordion, it appears, to overcome setting `height: 0` bye calculating heights for the expanded accordions - might need to test and verify. If so, it would be more JavaScript dependence to make the accordions expand (#1121 & #1122) - unlike the `display: block` route. See [lines 90 through 122](https://github.com/frend/frend.co/blob/gh-pages/_components/accordion/accordion.js).